tools/clang/lib/SPIRV/GlPerVertex.h

```diff
@@ -101,10 +102,14 @@ class GlPerVertex {
                    uint32_t semanticIndex, llvm::Optional<uint32_t> invocation,
                    uint32_t *value, bool noWriteBack);
 
+  /// Similar to tryToAccess, but only used for the PointSize builtin.
+  bool tryToAccessPointSize(hlsl::SigPoint::Kind sigPoint,
```

Comment:
Better to merge this into tryToAccess?

Reply:
It is not quite easy since there is no `Semantic::Kind` for `PointSize` at all. `PointSize` is a Vulkan-specific feature.